services/horizon/internal/actions_operation.go

```diff
@@ -76,7 +76,10 @@ func (action *OperationIndexAction) SSE(stream *sse.Stream) error {
 		func() {
 			stream.SetLimit(int(action.PagingParams.Limit))
 			operationRecords := action.OperationRecords[stream.SentCount():]
-			transactionRecords := action.TransactionRecords[stream.SentCount():]
+			var transactionRecords = action.TransactionRecords
```

Comment:
```suggestion
			var transactionRecords []history.Transaction
```
I think we should change this to above. In case there's a bug somewhere below (ex. no `IncludeTransactions` check) it will try to read from `nil` slice instead of something that have real values. It will crash but it's better then returning invalid results.